src/modules/app/components/side-nav/side-nav.styles.less

```diff
@@ -72,8 +72,11 @@
 
 @media @breakpoint-mobile {
   .SideNav {
+    display: flex;
+    flex-flow: column nowrap;
     height: 100vh;
     left: -12.5rem;
+    overflow: scroll;
```

Comment:
Update this to `auto`

Auto will only show the scrollbar when it’s needed, whereas scroll will always show it.

Reply:
Sorry, updated the comment prob after you saw it.  

Also update the property to `overflow-y`